Podfile

```diff
@@ -10,7 +10,9 @@ target 'Examples' do
 end
 
 target 'DocsCode' do
-  shared_pods
```

Comment:
We might as well remove `shared_pods`, now that it’s only used for the `Examples` target.

Probably should also switch the `Examples` target to use the non-symbols flavor of our SDK, that way we’re not downloading two different versions for this project.

Reply:
Thanks. 👍